### PR TITLE
artifactory-ha: SSL offload at Nginx service layer

### PR DESCRIPTION
**Pull request by Satheesh-Balachandran**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
SSL offload with the Nginx LoadBalancer in AWS

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
This can also be done by setting the artifactoryConf with the custom Server block. This will increase ease of configuring.

### CODE REVIEW

File: stable/artifactory-ha/values.yaml [hunk 1]
```diff
@@ -1245,8 +1247,12 @@ nginx:
         proxy_pass_header   Server;
         proxy_cookie_path   ~*^/.* /;
         proxy_pass          http://{{ include "artifactory-ha.fullname" . }}:{{ .Values.artifactory.externalPort }}/;
+        {{- if .Values.nginx.service.ssloffload}}
+        proxy_set_header    X-Artifactory-Override-Base-Url $http_x_forwarded_proto://$host/artifactory;
```
Comment (eldada) [inline stable/artifactory-ha/values.yaml]: Need to change to the new header: `X-JFrog-Override-Base-Url` and also remove the `/artifactory`.

**Comment by eldada** [comment]
Can you also add these changes to `artifactory` chart?

File: stable/artifactory-ha/values.yaml (hunk 1 above)
Comment (Satheesh-Balachandran) [reply stable/artifactory-ha/values.yaml]: Changed the header and removed the context

**Comment by Satheesh-Balachandran** [comment]
@eldada  Also merged the changes to Artifactory charts

**Comment by eldada** [comment]
Check failing CI

**Comment by Satheesh-Balachandran** [comment]
@eldada Thanks. Fixed it.

**Comment by Satheesh-Balachandran** [comment]
@danielezer @rimusz @chukka please review

**Pull request merged**